### PR TITLE
[improvement] Configurability + remove redundancy

### PR DESCRIPTION
**Pull request by atb00ker**
- Add sentry for django.
- Add leftlet configuration options.
- Moved common environment variables to openwisp base image.
- Moved postfix "init_command" script with the init_command script of other 
services to have a common script for all. (just following the pattern of other services.)

### CODE REVIEW

File: build/common/openwisp/settings.py [hunk 1]
```diff
@@ -181,6 +194,18 @@
...
 }
 
+# Sentry
+# https://sentry.io/for/django/
+
+if os.environ['DJANGO_SENTRY_DSN']:
+    import sentry_sdk
+    from sentry_sdk.integrations.django import DjangoIntegration
+    sentry_sdk.init(dsn=os.environ['DJANGO_SENTRY_DSN'],
+                    integrations=[DjangoIntegration()])
```
Comment (nemesisdesign) [inline build/common/openwisp/settings.py]: :+1:

File: build/common/openwisp/utils.py [hunk 2]
```diff
@@ -11,6 +11,11 @@ def filter(self, record):
         record.host = socket.gethostname()
         return True
 
+def env_bool(env):
+    if env in ["True", "true", "TRUE"]:
```
Comment (nemesisdesign) [inline build/common/openwisp/utils.py]: what about:

```python
return env in ["True", "true", "TRUE"]
```

File: build/common/utils.sh [hunk 3]
```diff
@@ -66,3 +68,83 @@ function envsubst_create_config {
         envsubst < ${1} > /etc/nginx/conf.d/${application}.${2}.conf
     done
 }
+
+function postfix_config {
```
Comment (nemesisdesign) [inline build/common/utils.sh]: is it possible to disable this container in case we use an external SMTP? Ensure it's explained in the README how to do this.

File: build/openwisp_websocket/daphne.conf [hunk 4]
```diff
@@ -14,3 +14,5 @@ autorestart=true
 stopsignal=INT
 stdout_logfile=/dev/stdout
 stderr_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile_maxbytes=0
```
Comment (nemesisdesign) [inline build/openwisp_websocket/daphne.conf]: why are these needed? If there's a reason, we may need to add a comment because it's not clear to me

File: build/openwisp_websocket/worker.conf [hunk 5]
```diff
@@ -14,3 +14,5 @@ autorestart=true
 stopsignal=INT
 stdout_logfile=/dev/stdout
 stderr_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
```
Comment (nemesisdesign) [inline build/openwisp_websocket/worker.conf]: same here

File: build/openwisp_websocket/daphne.conf (hunk 4 above)
Comment (atb00ker) [reply build/openwisp_websocket/daphne.conf]: It's required because otherwise we get `error: invalid seek`, it's not a fatal error that stops the service but it ruins the logs.
I will add a comment here. :smile:

File: build/common/utils.sh (hunk 3 above)
Comment (atb00ker) [reply build/common/utils.sh]: Added a note in the README. :-)

File: build/openwisp_websocket/daphne.conf (hunk 4 above)
Comment (nemesisdesign) [reply build/openwisp_websocket/daphne.conf]: ok, let me know when is done

Comment (atb00ker) [reply build/openwisp_websocket/daphne.conf]: Added a commit above this line. :smile:

File: build/common/utils.sh (hunk 3 above)
Comment (atb00ker) [reply build/common/utils.sh]: Added a section in README.
